opencti-platform/opencti-front/src/components/dashboard/WidgetPolarArea.tsx

```diff
@@ -4,29 +4,38 @@ import { useTheme } from '@mui/styles';
 import { ApexOptions } from 'apexcharts';
 import { polarAreaChartOptions } from '../../utils/Charts';
 import type { Theme } from '../Theme';
+import useDistributionGraphData from '../../utils/hooks/useDistributionGraphData';
 
 interface WidgetPolarAreaProps {
-  data: {
-    label: string,
-    value: number
-  }[]
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any[]
```

Comment:
I'm curious; does it work if you use DistributionQueryData from useDistributionGraphData ?

Reply:
I'll check thanks!

Reply:
Done

Reply:
great !